stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -73,6 +78,9 @@ public DOM(DOMProvider.Factory providerFactory) {
 
     mObjectMapper = new ObjectMapper();
     mObjectIdMapper = new DOMObjectIdMapper();
+
+    mSearchResults = Collections.synchronizedMap(
+      new HashMap<String, List<Integer>>());
```

Comment:
Rick specified that it should be this way. Are you sure? ref: https://github.com/facebook/stetho/pull/225#discussion_r34513807